README.adoc

```diff
@@ -1833,6 +1833,60 @@ storages:
     url: https://exstorage2/api/webdav
 ----
 
+=== Multi-Fairspace metadata views integration
+
+Fairspace can have only single data model configured. However, if there is a need to have data from multiple models,
+representing multiple domains that would be combined combined in single user interface, it is possible to integrate multiple Fairspace instances together.
+
+
+Fairspace (main) instance can be configured as a view interface of metadata from another (external) Fairspace instance.
+External Fairspace can have a different metadata model configured than the main Fairspace.
+Important precondition is that the external instance has to be connected to the same Keycloak realm as the main instance.
+
+
+External metadata can be searched and browsed in the same way as the internal metadata views. When configured, there is an extra page,
```

Comment:
do you think it makes sense to add a picture of the interface for a few models here or at the beginning of the doc?

Reply:
Not sure... we don't have a proper demo instance to take the screenshot of...